packages/graphql-plugin-auth/src/JWTPlugin.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import jsonwebtoken from "jsonwebtoken";
+import Debug from "debug";
+import { DEBUG_PREFIX } from "./constants";
+
+const debug = Debug(DEBUG_PREFIX);
+
+export interface JWTPluginInput {
+    secret: string;
+    noVerify?: boolean;
+    rolesPath?: string;
+}
+
+class JWTPlugin {
```

Comment:
**question:** should we think carefully about the naming of these classes before committing them forever to the API? Personally, I had something like `Neo4jGraphQLPluginAuthJWT` in mind, however, I can see why some would see that as quite verbose. I just feel like all plugins (realise it's only two right now) should have a common prefix.

Reply:
Personally, I think `Neo4jGraphQLAuthJWTPlugin` reads a bit better

Reply:
I think I'm siding with readability on this one, so `Neo4jGraphQLAuthJWTPlugin`.

Reply:
Happy to go with this, which would establish a convention of `Neo4jGraphQL<PluginSlotName><SpecificPluginName>Plugin` for plugins.

Reply:
I'm happy with this, ill make the changes and change the `jwt` slot to `auth`.